cuda-flags.file

```diff
@@ -2,8 +2,8 @@
 ## INCLUDE cpp-standard
 # define the CUDA compilation flags in a way that can be shared by SCRAM-based and regular tools
 
-# build support for Pascal (6.x), Volta (7.0), Turing (7.5), Ampere (8.x) and Lovelace (8.9)
-%define cuda_arch 60 70 75 80 89
+# build support for Pascal (6.x), Volta (7.0), Turing (7.5), Ampere (8.x), Lovelace (8.9) and Hopper (9.0)
+%define cuda_arch 60 70 75 80 89 90
```

Comment:
@fwyzard , any objection on added `90` cuda arch? Can we drop any of the existing archs e.g. `60`?

Reply:
@fwyzard , any objection? I am fine with building for cuda_arch 90